setup.py

```diff
@@ -54,5 +56,7 @@
     keywords='libheif, heif, heic, high-efficiency image format',
     python_requires='>=3.6',
     ext_modules=extensions,
-    packages=['cyheifloader']
+    packages=['cyheifloader'],
+    install_requires=['pillow'],
+    data_files=[('bin', [str(x.absolute()) for x in pathlib.Path(lib_dirs[0]).glob("*.dll")])] if sys.platform.startswith('win32') else [],

```

Comment:
Since vcpkg will install every package's binaries to the same folder, it would be better to explicitly just pull heif.dll, libde265.dll and libx265.dll for now.